client/src/routes/ProtectedRoutes.js

```diff
@@ -0,0 +1,26 @@
+import React, { useContext } from "react";
+import { Redirect, Route } from "react-router-dom";
+import { userState } from "../provider/UserContext";
+
+const ProtectedRoutes = ({ component: Component, ...rest }) => {
+  const {
+    state: { isAuthenticated },
+  } = useContext(userState);
+  return (
+    <Route
```

Comment:
Can you use a wrapper like this for signup/login as well? It might be cleaner in the future if there's a decision made to make explore the default landing page for example

Reply:
If you're ok with this, make a new issue and you can work on it at your own discretion

Reply:
yeah sure, i can look into this